packages/http-client/src/auth.ts

```diff
@@ -38,26 +38,25 @@ export class BearerCredentialHandler implements ifm.IRequestHandler {
 
   // currently implements pre-authorization
   // TODO: support preAuth = false where it hooks on 401
-  prepareRequest(options: any): void {
-    options.headers['Authorization'] = 'Bearer ' + this.token
+  prepareRequest(options: http.RequestOptions): void {
+    if (!options.headers) {
+      throw Error('The request has no headers')
+    }
+    options.headers['Authorization'] = `Bearer ${this.token}`
   }
 
   // This handler cannot handle 401
-  canHandleAuthentication(response: ifm.IHttpClientResponse): boolean {
+  canHandleAuthentication(): boolean {
```

Comment:
The linter errors on unused parameters.  TypeScript still considers this as implementing the interface method.